activiti-services/activiti-services-connectors/src/main/java/org/activiti/services/connectors/channel/ServiceTaskChannels.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.services.connectors.channel;
+
+import org.springframework.cloud.stream.annotation.Input;
+import org.springframework.cloud.stream.annotation.Output;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.messaging.SubscribableChannel;
+
+public interface ServiceTaskChannels {
+
+    String SERVICE_TASK_RESULT_CONSUMER = "serviceTaskResultConsumer";
+
+    @Input(SERVICE_TASK_RESULT_CONSUMER)
```

Comment:
@erdemedeiros good stuff.. but let's rename this to integrationEventsProducer and integrationResultsConsumer.